map_model/src/pathfind/v2.rs

```diff
@@ -158,6 +158,9 @@ impl PathV2 {
             // At the simulation layer, we may need to block intermediate lanes to exit a driveway,
             // so reflect that cost here. The high cost should only be worth it when the v2 path
             // requires that up-front turn from certain lanes.
+            //
+            // TODO This is only valid if we were leaving from a driveway! This is making some
```

Comment:
This crashes SP center. The pathfinding layer always assumes the start lane can change, which is just super untrue! Need to revisit when I introduced this code and see if I thought about it then.